notebooks/usage.ipynb

```diff
@@ -43,36 +43,70 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "sys.path.append(str(Path.home() / Path(\"Development\") / \"iSparrow\"))"
```

Comment:
replaced it with a relative include of the package. Since this is only used for the setup method, maybe the solution would be to have a setup module in the src folder? Would add this in a separate PR that does the packaging and install stuff?